staging/cse/windows/azurecnifunc.tests.suites/AzureCNI.Default.OverlayDualStack.conflist

```diff
@@ -0,0 +1,52 @@
+{
+  "cniVersion": "0.3.0",
+  "name": "azure",
+  "adapterName": "",
+  "plugins": [
+    {
+      "type": "azure-vnet",
+      "mode": "bridge",
+      "bridge": "azure0",
+      "executionMode": "v4swift",
```

Comment:
checking on this

Reply:
it would still be good to know but we don't update this value in AB so for testing purposes it isn't a big concern. We're more concerned with making sure the right exception list written